### PR TITLE
TW-601 feat: new encrypted message design

### PR DESCRIPTION
**Pull request by imGok**
#601 
Design 

image

Demo : 

image

https://github.com/linagora/twake-on-matrix/assets/48354990/9cd27408-86fe-422c-a737-8ecf92a2e167

### CODE REVIEW

File: lib/pages/chat/events/encrypted_content.dart [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+                ),
+              ],
+            ),
+          ],
+        ),
+      ),
+    );
+  }
+
+  void _verifyOrRequestKey(BuildContext context) async {
```
Comment (drminh2807) [inline lib/pages/chat/events/encrypted_content.dart:58]: you should move it to ChatController

**Comment by drminh2807** [comment]
271674637-d6bd989a-f9f9-4c84-9a63-ac7053970059
I think they should be in 3 columns

File: lib/pages/chat/events/encrypted_content.dart (hunk 1 above)
Comment (sherlockvn) [reply lib/pages/chat/events/encrypted_content.dart:58]: or mixin

Comment (sherlockvn) [reply lib/pages/chat/events/encrypted_content.dart:58]: because it include both logic and view

**Comment by imGok** [comment]
> 271674637-d6bd989a-f9f9-4c84-9a63-ac7053970059 I think they should be in 3 columns

layout is not like this anymore it's 2 columns

File: lib/pages/chat/events/encrypted_content.dart (hunk 1 above)
Comment (imGok) [reply lib/pages/chat/events/encrypted_content.dart:58]: you are right, I created a mixin

**Review by sherlockvn — APPROVED** [review]

File: lib/pages/chat/events/message.dart [hunk 2]
```diff
@@ -211,6 +212,13 @@ class Message extends StatelessWidget {
                                             : FutureBuilder<User?>(
                                                 future: event.fetchSenderUser(),
                                                 builder: (context, snapshot) {
+                                                  // Doesn't show the sender name if the event is encrypted.
+                                                  if (event.type ==
+                                                      EventTypes.Encrypted) {
+                                                    return const SizedBox
+                                                        .shrink();
+                                                  }
```
Comment (hoangdat) [inline lib/pages/chat/events/message.dart:220]: please wait #719 resolve and applied the same

**Review by drminh2807 — APPROVED** [review]

File: lib/pages/chat/events/message.dart (hunk 2 above)
Comment (imGok) [reply lib/pages/chat/events/message.dart]: done

**Review by hoangdat — APPROVED** [review]

**Review by nqhhdev — APPROVED** [review]

**Pull request merged**